forch_interface.py

```diff
@@ -0,0 +1,167 @@
+#!/Users/ayush/Desktop/core/nucleus-workflows/.venv/bin/python
+
+import asyncio
+import json
+import sys
+from dataclasses import dataclass
+from typing import Literal
+
+from latch_config.config import PostgresConnectionConfig, read_config
+from latch_data_validation.data_validation import untraced_validate
+from latch_postgres.postgres import (
+    LatchAsyncConnection,
+    get_pool,
+    get_with_conn_retry,
+    sqlq,
+)
+
+config = read_config(PostgresConnectionConfig, "db_")
+
+pool = get_pool(config, "nextflow_forch_test", read_only=False)
+with_conn_retry = get_with_conn_retry(pool, config)
+
+
+@dataclass
+class TaskInfo:
+    id: int
+
+
+@dataclass
+class CreateTaskInput:
+    display_name: str
+    container_image: str
+    container_entrypoint: list[str]
+    cpus: int
+    memory_bytes: int
+    gpu_type: str | None
+    gpus: int
+
+
+async def create_task(payload: CreateTaskInput):
```

Comment:
u can just wrap this function in `with_conn_retry` without a helper, since it doesn't do anything else